packages/main/src/plugin/kubernetes-client.ts

```diff
@@ -168,7 +168,9 @@ export class KubernetesClient {
       // check if path exists
       if (existsSync(userKubeconfigPath)) {
         this.kubeconfigPath = userKubeconfigPath;
-        await this.refresh();
+        this.refresh()
+          .then(() => console.log('Refresh of kube resources on startup complete'))
```

Comment:
Not sure if we need a trace when it completes

Reply:
LMK if you need this addressed. I believe the sign-off is all good now 👍